BUILD

```diff
@@ -1,4 +1,13 @@
+python_requirement(
+    name="jmespath",
+    requirements=["jmespath"],
+)
+
 python_requirements(
     name="root",
     source="pyproject.toml",
+    overrides={
```

Comment:
This is an interesting problem I came across. aws-lambda-powertools does not declare jmespath as a dependency since it is included in lambda runtime.